qemu/tests/smbios_table.py

```diff
@@ -56,6 +56,11 @@ def run(test, params, env):
         qemu_binary = utils_misc.get_qemu_binary(params)
         tmp = utils_misc.get_support_machine_type(qemu_binary)
         (support_machine_types, expect_system_versions) = tmp
+        # remove alias machine types to prevent duplication
+        for version in expect_system_versions:
+            if 'alias of' in version:
+                index = expect_system_versions.index(version)
+                del support_machine_types[index]
```

Comment:
`utils_misc.get_support_machine_type`  already exclude the "alias of".   it use:
```
2115     for item in o[1:]: 
```

Reply:
Thanks for pointing it out. This is correct. I will create a new patch.

Reply:
On a second thought, I want to know if the "for item in o[1:]: " is the right way to do? What if the machine type list doesn't have "alias" define? Does it mean we will miss testing one machine type?

-Wei

Reply:
@huangwei  yes, your are right, this design base on the assumption that the fist machine type is an alias. It is not a strict logical even if it can work now,

I think we can improve "utils_misc.get_support_machine_type" , what do you think ?

Reply:
I think we should improve it. IMO we shouldn't make the assumption about the existence of alias type.

Reply:
@huangwei  i am sorry i make a mistake before. In utils_misc.get_support_machine_type , "alias of" doesn't be exclude, it exclude the first line : "Supported machines are:"
`2115     for item in o[1:]: ` 
I have send a new patch to exclude "alias of" in utils_misc.get_support_machine_type(), it can help avoid the problem you mentioned.